src/test/java/LoginTests.java

```diff
@@ -1,141 +1,160 @@
-
-    }
-
-    @Test(enabled = false)
-    public void LoginValidEmailEmptyPasswordTest () {
-        WebDriver driver;
-        driver = new ChromeDriver();
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-
-        String url = "https://bbb.testpro.io/";
-        driver.get(url);
-
-        WebElement emailField = driver.findElement(By.cssSelector("[type='email']"));
-        emailField.click();
-        emailField.sendKeys("demo@class.com");
+        wait.until(ExpectedConditions.textToBePresentInElement(driver.findElement(playlistLocator), "PLAYLISTS"));*/
+        @Test(enabled = true)
+        public void LoginValidEmailValidPasswordTest () {
+            launchBrowser("https://bbb.testpro.io");
+            LoginPage loginPage = new LoginPage(driver);
+            HomePage homePage = new HomePage(driver);
+            loginPage.provideEmail("nargiza10041@gmail.com");
+            loginPage.providePassword("te$t$tudent");
+            loginPage.clickSubmitBtn();
+            Assert.assertTrue(homePage.getInfoAboutKoel().isDisplayed());
+        }
+        @Test
+        public void InfoKoelPage () {
+            AboutKoelPage infoPage = new AboutKoelPage(driver);
+            infoPage.login();
+            infoPage.clickButtonInfo();
+            infoPage.clickCloseButton();
+        }
+
+        @Test(enabled = false)
+        public void LoginInvalidEmailPasswordTest () throws InterruptedException {
+            provideEmail("dem@class.com");
+            providePassword("");
+            clickSubmitBtn();
+
+            // Vd
+            Thread.sleep(2000);
+            Assert.assertEquals(driver.getCurrentUrl(), url);
+
+        }
+
+        @Test(enabled = false)
+        public void LoginValidEmailEmptyPasswordTest () {
+            WebDriver driver;
+            driver = new ChromeDriver();
+            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+            String url = "https://bbb.testpro.io/";
+            driver.get(url);
+
+            WebElement emailField = driver.findElement(By.cssSelector("[type='email']"));
+            emailField.click();
+            emailField.sendKeys("demo@class.com");
+
+            WebElement passwordField = driver.findElement(By.cssSelector("[type='password']"));
+            passwordField.click();
+
+            WebElement submitButton = driver.findElement(By.cssSelector("[type='submit']"));
+            submitButton.click();
+
+            WebElement avatarIcon = driver.findElement(By.cssSelector("[alt='Avatar of student']"));
+            Assert.assertTrue(avatarIcon.isDisplayed());
+
+            driver.quit();
+        }
+
+        @Test(enabled = false)
+        public void SwitchToSongsMenu () {
+            WebDriver driver;
+            driver = new ChromeDriver();
+            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+            String url = "https://bbb.testpro.io/";
```

Comment:
it is necessary to make this variable global so as not to prescribe it in each method